.github/workflows/build.yml

```diff
@@ -9,7 +9,7 @@ on:
       - main
 
 env:
-  IMAGE_NAME: ghcr.io/renovatebot/renovate-approve-bot-bitbucket-cloud
+  IMAGE_NAME: ghcr.io/renovatebot/renovate-approve-bot-bitbucket
```

Comment:
@maxbrunet @viceice do you have any concerns about this breaking change? Ideas on how to unbreak it for users, or alert them?

Reply:
no way to unbreak.

Reply:
we probably should publish both images for a while

Reply:
I wonder if we should bundle other breaking changes for v2?

* #166
* Migrate to ESM with upgrade to got v12 and jest v28

And the code getting larger, TypeScript (#58) starts making more sense

FYI I do not use Bitbucket anymore, so I will not be able to help testing this